test/controllers/changesets_controller_test.rb

```diff
@@ -121,8 +125,9 @@ def test_index_bbox
   # Checks the display of the user changesets listing
   def test_index_user
     user = create(:user)
-    create(:changeset, :user => user)
-    create(:changeset, :closed, :user => user)
+    create(:changeset, :user => user, :num_changes => 1)
+    create(:changeset, :closed, :user => user, :num_changes => 1)
```

Comment:
Why does this need changes when neither the `index` action or the rest of this test has changed?

Reply:
see below